### PR TITLE
Full metronome control

### PR DESCRIPTION
**Pull request by creaktive**
UI elements mapped bidirectionally to all the features that PianoTeq built-in metronome provides (tempo, time signature, volume, first beat accent). And the start/stop button, of course :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -105,17 +105,57 @@
...
+              <option value="7">+7 dB</option>
+              <option value="10">+10 dB</option>
+              <option value="15">+15 dB</option>
+              <option value="20">+20 dB</option>
+            </select>
+            <label for="metronome-accent" id="metronome-accent-label">
+              <input type="checkbox" id="metronome-accent" />
+              Accent on first beat
+            </label>
             <button class="pure-button" id="metronome-toggle">Start</button>
```
Comment (shuhaowu) [inline index.html]:
```suggestion
            <button class="pure-button" id="metronome-toggle">Toggle Metronome</button>
```

File: index.html [hunk 2]
```diff
@@ -105,17 +105,57 @@
         </div>
       </div>
 
-      <!--
```
Comment (shuhaowu) [inline index.html:108]: Can you add a `<hr />` here to delimit the UI a bit better?

**Review by shuhaowu — COMMENTED** [review]
I noticed that the UI is not set to disabled if an error occurs. I don't have time to look at this problem very carefully right now. Do you think you can take a look and fix it? Once it's done (and a few small suggestions) I think this can be merged.

image

**Comment by creaktive** [comment]
Thanks for the review! Fixed the UI bug and implemented the suggestions.

**Comment by shuhaowu** [comment]
Thanks for all this!

**Comment by creaktive** [comment]
Always a pleasure! Kudos for awesome and useful project.

**Pull request merged**